github/ci/prow/files/jobs/kubevirt.github.io/kubevirt-github-io-periodics.yaml

```diff
@@ -0,0 +1,14 @@
+periodics:
+  - interval: 24h
+    name: kubevirt-io-periodic-link-checker
+    decorate: true
+    spec:
+      containers:
+        - image: docker.io/library/ruby
+          env:
+            - name: NOKOGIRI_USE_SYSTEM_LIBRARIES
+              value: "true"
+          command: ["/bin/sh", "-c"]
+          args: ["git clone https://github.com/kubevirt/kubevirt.github.io.git && cd kubevirt.github.io && /usr/local/bin/bundle install && bundle exec rake"]
```

Comment:
What is the periodic job doing?

Reply:
The periodic jo will download all the links that website has, including blogs and so on with a pace of 24h every check

Reply:
ok, so it acts as a warning? How do you get notified if it fails?

Reply:
Yeah, sometimes we get errors on blog links that are broken because they points to external resources. If there is any 4XX or 5XX error on any link the tests will fail and send us an email :). 

Also we added us as a reviewers/approvers in order to check the PRs

But as you said over Slack, crier will be a great option

Reply:
I will test crier on my stagging platform, for now I've asked for a Slack API token to allow the notifications :), meanwhile I will take a look manually. There is not so much PR/MR